modules/installation/pages/hardware-requirements.adoc

```diff
@@ -29,3 +36,20 @@ For installation on {ibmz}, see xref:installation:install-ibmz.adoc[].
 |                          | [path]``/var/lib/pgsql`` Minimum 50{nbsp}GB
 |                          | [path]``/var/spacewalk`` Minimum 50{nbsp}GB per SUSE product and 360{nbsp}GB per Red Hat product
 |===
+
+
+
+== Proxy Hardware Requirements
+
+[cols="1,1", options="header"]
+.Proxy Hardware Requirements
+|===
+| Hardware                 | Recommended
+| CPU                      | Minimum 2 dedicated 64-bit CPU cores
+| RAM:                     | _Test Server_ Minimum 2{nbsp}GB
+|                          | _Base Installation_ Minimum 16{nbsp}GB
+|                          | _Production Server_ Minimum 8{nbsp}GB
```

Comment:
I did not have "Base Installation" in my proposal as I thought it does not make sense.
If you want to have it, please put it on 4GB, not more than "Production" .